Training_25/Program.cs

```diff
@@ -3,11 +3,26 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program on T06 branch. Assignment Q6: Digital Root.
 // ------------------------------------------------------------------------------------------------
+using static System.Console;
 namespace Training_25;
+
 internal class Program {
    static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+      WriteLine ("Enter a number, to find it's digital root:");

```

Comment:
```suggestion
      WriteLine ("Enter a number, to find it's digital root: ");
```
Please add a space after the colon in the prompt string. This improves readability and provides a cleaner user input experience.